NuGet.Config

```diff
@@ -2,8 +2,7 @@
 <configuration>
   <packageSources>
     <clear/>
-    <add key="CI Builds (aspnetvnext)" value="https://www.myget.org/F/aspnetvnext/api/v3/index.json" />
-    <add key="CI Builds (aspnetcidev)" value="http://myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="CI Builds (aspnetvnext)" value="https://www.myget.org/F/aspnetrelease/api/v3/index.json" />
```

Comment:
Would it a good idea for this repo to just use the cli-deps feed?  https://dotnet.myget.org/F/cli-deps/api/v3/index.json